examples/preapproval/search/search_subscription_payment_orders.rb

```diff
@@ -1,7 +1,7 @@
 require_relative '../../boot'
 
 code = 'CODE'
-status = :status # Are availables: [ :scheduled, :processing, :not_processed, :suspended, :paid, :not_paid ]
+status = :processing # Are availables: [ :scheduled, :processing, :not_processed, :suspended, :paid, :not_paid ]
```

Comment:
`are availables`, I think `there are…` or `the available status are`… I mean the subject is missing :P